README.md

```diff
@@ -75,6 +74,10 @@ There is only an `Octo <object> <action> [arguments]` command:
 | card | add | |
 | | delete | |
 | | move | |
+| review| start| Start a new review |
+| | comments| View in-progress review comments |
+| | submit| Submit the review |
```

Comment:
test

Reply:
another comment

Reply:
someone is watching 👀 😆

Reply:
Can you see this live in Octo while the PR buffer is open? :thinking: :scream:

Reply:
no, review comments are not shown in the PR buffer, only the top-level final review comment. Im doing it that way because the web view is to bloatered with comments in my opinion.
the way to see review comments in octo is with `Octo review threads`

Reply:
Sorry to abuse this thread, but what do you think about folds?

Reply:
no problem as long as I get notifications :)

re: folds

check: https://github.com/pwntester/octo.nvim/issues/80
folds may introduce two benefits: 1) folding xD and 2) replacing the current sign column
The later is not possible yet, but I may introduce folds later, specially if new APIs are exposed for them

re: floating windows
check: https://github.com/pwntester/octo.nvim/issues/59
I dont really see use cases for them, but if you have something specific on mind, please comment on that issue